### PR TITLE
feat: Vonage API

### PR DESCRIPTION
**Pull request by rahmat-st**

### CODE REVIEW

**Review by TheLearneer — COMMENTED** [review]
The Vonage API seems to not have any free plan (not talking about demos).
As of now, this API list contains only the APIs that have free plans.

@rahmat-st can you please confirm it the API has only paid plans?

**Comment by rahmat-st** [comment]
Do you mean the demo version is like a few days trial period?

I think Vonage is similar to Stripe. Vonage provides a free version for the sandbox/testing environment. If Stripe charges a fee for each transaction, Vonage is the same, you will pay according to the service you use in production.

https://www.vonage.com/communications-apis/pricing/

**Comment by TheLearneer** [comment]
That makes sense @rahmat-st 

However please make sure to comply with all the requirement mentioned in #2 for the PR to be accepted as part of Hacktoberfest 🙏

**Comment by rahmat-st** [comment]
No problem, I already have 4 or more PRs for hacktoberfest btw

**Review by TheLearneer — APPROVED** [review]

File: src/content/apis/development/vonage.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+---
+name: Vonage
+description: Integrate sms, voice, video and two-factor authentication into your apps with Vonage communication APIs.
+categories:
+  - development
+features:
+  https: true
```
Comment (TheLearneer) [inline src/content/apis/development/vonage.md:7]:
```suggestion
  https: true
  pricing: paid
```

**Review by TheLearneer — CHANGES_REQUESTED** [review]
@rahmat-st Please make this change and we can proceed with merging this PR 🙏

**Pull request closed without merging**